popup.js

```diff
@@ -1,150 +1,8 @@
-  formatDate:'d.m.Y',
-    // TODO: Ideally, `defaultDate` should be today's date 
-    defaultDate:'+10.04.1996', 
-  defaultTime:'00:00',
-  timepickerScrollbar:false
-});
-
-
-$('#datetimepicker10').datetimepicker({
-  step:5,
-  inline:true
-});
-$('#datetimepicker_mask').datetimepicker({
-  mask:'9999/19/39 29:59'
-});
-
-$('#datetimepicker1').datetimepicker({
-  datepicker:false,
-  format:'H:i',
-  step:5
-});
-$('#datetimepicker2').datetimepicker({
-  yearOffset:222,
-  lang:'ch',
-  timepicker:false,
-  format:'d/m/Y',
-  formatDate:'Y/m/d',
-  minDate:'-1970/01/02', // yesterday is minimum date
-  maxDate:'+1970/01/02' // and tommorow is maximum date calendar
-});
-$('#datetimepicker3').datetimepicker({
-  inline:true
-});
-$('#datetimepicker4').datetimepicker();
-$('#open').click(function(){
-  $('#datetimepicker4').datetimepicker('show');
-});
-$('#close').click(function(){
-  $('#datetimepicker4').datetimepicker('hide');
-});
-$('#reset').click(function(){
-  $('#datetimepicker4').datetimepicker('reset');
-});
-$('#datetimepicker5').datetimepicker({
-  datepicker:false,
-  allowTimes:['12:00','13:00','15:00','17:00','17:05','17:20','19:00','20:00'],
-  step:5
-});
-$('#datetimepicker6').datetimepicker();
-$('#destroy').click(function(){
-  if( $('#datetimepicker6').data('xdsoft_datetimepicker') ){
-    $('#datetimepicker6').datetimepicker('destroy');
-    this.value = 'create';
-  }else{
-    $('#datetimepicker6').datetimepicker();
-    this.value = 'destroy';
-  }
-});
-var logic = function( currentDateTime ){
-  if (currentDateTime && currentDateTime.getDay() == 6){
-    this.setOptions({
-      minTime:'11:00'
-    });
-  }else
-    this.setOptions({
-      minTime:'8:00'
-    });
-};
-$('#datetimepicker7').datetimepicker({
-  onChangeDateTime:logic,
-  onShow:logic
-});
-$('#datetimepicker8').datetimepicker({
-  onGenerate:function( ct ){
-    $(this).find('.xdsoft_date')
-      .toggleClass('xdsoft_disabled');
-  },
-  minDate:'-1970/01/2',
-  maxDate:'+1970/01/2',
-  timepicker:false
-});
-$('#datetimepicker9').datetimepicker({
-  onGenerate:function( ct ){
-    $(this).find('.xdsoft_date.xdsoft_weekend')
-      .addClass('xdsoft_disabled');
-  },
-  weekends:['01.01.2014','02.01.2014','03.01.2014','04.01.2014','05.01.2014','06.01.2014'],
-  timepicker:false
-});
-var dateToDisable = new Date();
-  dateToDisable.setDate(dateToDisable.getDate() + 2);
-$('#datetimepicker11').datetimepicker({
-  beforeShowDay: function(date) {
-    if (date.getMonth() == dateToDisable.getMonth() && date.getDate() == dateToDisable.getDate()) {
-      return [false, ""]
-    }
-
-    return [true, ""];
-  }
-});
-$('#datetimepicker12').datetimepicker({
-  beforeShowDay: function(date) {
-    if (date.getMonth() == dateToDisable.getMonth() && date.getDate() == dateToDisable.getDate()) {
-      return [true, "custom-date-style"];
-    }
-
-    return [true, ""];
-  }
+disabledDates:['1986/01/08','1986/01/09','1986/01/10'],
+startDate:  '1986/01/05'
```

Comment:
The `startdate` should be today's date !

Remove `disabledDates`.
Dont fiddle with the commented out code.
Read them and understand their relevance